config/locales/simple_form.en.yml

```diff
@@ -0,0 +1,5 @@
+en:
```

Comment:
I don't feel as strongly here, but part of me says it's easier to find these if they're all in `hyrax.en.yml`

Reply:
Yeah, I've been putting config since this in `hyrax.en.yml` I can move.